osbuild/main_cli.py

```diff
@@ -168,7 +172,15 @@ def osbuild_cli():
             if r["success"] and exports:
                 for pid in exports:
                     export(pid, output_directory, object_store, manifest)
-
+    except OSbuildError as e:
+        if args.json:
+            r = format_exception(e)
```

Comment:
Even, if for now, we sure the code between formats, let's put the infrastructure in place, i.e. let's go via the format specific code.

Reply:
Sure. I kept formats_common.py as it is defines the useful type OSBuildError. I added a format method in it. Which is used to format the return value for either v1 and v2.
The function in the format is just:

```
def format_osbuild_error(error: OSBuildError):
    return error.format()
```

We can later change this if you prefer